mteb/models/model2vec_models.py

```diff
@@ -0,0 +1,209 @@
+    open_weights=True,
+    revision="5f4f5ca159b7321a8b39739bba0794fa0debddf4",
+    release_date="2024-10-14",  # blog post date
+    n_parameters=103 * 1e6,
+    max_tokens=1000000,  # I took this from the current leaderboard, I'm not sure if it's unlimited in theory, tough
+    embed_dim=256,
+    license="mit",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
+    reference="https://huggingface.co/minishlab/M2V_base_glove_subword",
+    use_instructions=False,
+    adapted_from="BAAI/bge-base-en-v1.5",
+    superseded_by=None,
+)
+
+
+m2v_base_glove = ModelMeta(
+    loader=partial(
+        Model2VecWrapper,
+        model_name="minishlab/M2V_base_glove",
+    ),
+    name="minishlab/M2V_base_glove",
+    languages=["eng_Latn"],
+    open_weights=True,
+    revision="38ebd7f10f71e67fa8db898290f92b82e9cfff2b",
+    release_date="2024-10-14",  # blog post date
+    n_parameters=102 * 1e6,
+    max_tokens=1000000,
+    embed_dim=256,
+    license="mit",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
+    reference="https://huggingface.co/minishlab/M2V_base_glove",
+    use_instructions=False,
+    adapted_from="BAAI/bge-base-en-v1.5",
+    superseded_by=None,
+)
+
+m2v_base_output = ModelMeta(
+    loader=partial(
+        Model2VecWrapper,
+        model_name="minishlab/M2V_base_output",
+    ),
+    name="minishlab/M2V_base_output",
+    languages=["eng_Latn"],
+    open_weights=True,
+    revision="02460ae401a22b09d2c6652e23371398329551e2",
+    release_date="2024-10-14",  # blog post date
+    n_parameters=7.56 * 1e6,
+    max_tokens=1000000,
+    embed_dim=256,
+    license="mit",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
+    reference="https://huggingface.co/minishlab/M2V_base_output",
+    use_instructions=False,
+    adapted_from="BAAI/bge-base-en-v1.5",
+    superseded_by=None,
+)
+
+m2v_multilingual_output = ModelMeta(
+    loader=partial(
+        Model2VecWrapper,
+        model_name="minishlab/M2V_multilingual_output",
+    ),
+    name="minishlab/M2V_multilingual_output",
+    languages=["eng_Latn"],
+    open_weights=True,
+    revision="2cf4ec4e1f51aeca6c55cf9b93097d00711a6305",
+    release_date="2024-10-14",  # blog post date
+    n_parameters=128 * 1e6,
+    max_tokens=1000000,
+    embed_dim=256,
+    license="mit",
+    similarity_fn_name="cosine",
+    framework=["PyTorch"],
+    reference="https://huggingface.co/minishlab/M2V_multilingual_output",
+    use_instructions=False,
+    adapted_from="sentence-transformers/LaBSE",
+    superseded_by=None,
+)
+
+potion_base_2m = ModelMeta(
+    loader=partial(
+        Model2VecWrapper,
+        model_name="minishlab/potion-base-2M",
+    ),
+    name="minishlab/potion-base-2M",
+    languages=["eng_Latn"],
+    open_weights=True,
+    revision="86db093558fbced2072b929eb1690bce5272bd4b",
+    release_date="2024-10-14",  # blog post date
+    n_parameters=2 * 1e6,
+    max_tokens=1000000,
+    embed_dim=256,
```

Comment:
Embed dim should be 64 for potion-base-2M:
```suggestion
    embed_dim=64,
```